easybuild/easyblocks/g/gromacs.py

```diff
@@ -161,10 +234,41 @@ def install_step(self):
                         self.lib_subdir = os.path.dirname(libpaths[0])[len(self.installdir)+1:]
                         self.log.info("Found lib subdirectory that contains %s: %s", libname, self.lib_subdir)
                         break
-
         if not self.lib_subdir:
             raise EasyBuildError("Failed to determine lib subdirectory in %s", self.installdir)
 
+        # Install a version with the MPI suffix
+        if self.toolchain.options.get('usempi', None):
+            if LooseVersion(self.version) < LooseVersion('4.6'):
+                cmd = "make distclean"
+                (out, _) = run_cmd(cmd, log_all=True, simple=False)
+                self.cfg.update('configopts', "--enable-mpi --program-suffix={0}".format(self.cfg['mpisuffix']))
+                ConfigureMake.configure_step(self)
+                self.cfg.update("buildopts", "mdrun")
+                super(EB_GROMACS, self).build_step()
+                cmd = "%s make install-mdrun %s" % (self.cfg['preinstallopts'], self.cfg['installopts'])
+                (out, _) = run_cmd(cmd, log_all=True, simple=False)
+            else:
+                cmake_objdir = "build-mdrun-only"
+                os.mkdir(cmake_objdir)
+                os.chdir(cmake_objdir)
+                self.cfg['configopts'] = re.sub(r'-DGMX_MPI=OFF', r'', self.cfg['configopts'])
+                if self.cfg['mpi_numprocs'] == 0:
+                    self.log.info("No specific number of test MPI tasks requested -- using parallelism ({0})".format(self.cfg['parallel']))
+                    self.cfg['mpi_numprocs'] = self.cfg['parallel']
+                elif self.cfg['mpi_numprocs'] > self.cfg['parallel']:
+                    self.log.warning("Number of test MPI tasks ({0}) is greater than parallelism ({1})".format(self.cfg['mpi_numprocs'], self.cfg['parallel']))
+                self.cfg.update('configopts', "-DGMX_MPI=ON -DGMX_THREAD_MPI=OFF -DMPIEXEC={0} -DMPIEXEC_NUMPROC_FLAG={1} -DNUMPROC={2} -DGMX_BUILD_MDRUN_ONLY=ON".format(self.cfg['mpiexec'], self.cfg['mpiexec_numproc_flag'], self.cfg['mpi_numprocs']))
```

Comment:
@valtandor why did you include `-DGMX_BUILD_MDRUN_ONLY=ON` here (and also add `mdrun` to `buildopts` above for older GROMACS versions)?

I had to get rid of this in #960 because I was resulting in an incomplete installation w.r.t. MPI, e.g. `libgromacs_mpi.a` was missing, etc.

It seems fine to just drop this, since the MPI-able binaries & libraries are suffixed with `_mpi`, so nothing is overwritten?